src/SourceBuild/tarball/content/repos/newtonsoft-json901.proj

```diff
@@ -0,0 +1,31 @@
+<Project>
```

Comment:
These _external_ repo projects will never get used and a source-build project will be needed instead right?  If not handled with this PR can you log an issue or better yet make note of it in the source-build onboard issue?

Reply:
Agreed.  I added this as the description on https://github.com/dotnet/source-build/issues/2273